ansible-inventory-diff.sh

```diff
@@ -45,4 +45,5 @@ mkdir /diff
 (cd a && ansible-inventory --list "$@" --output /diff/a.yml)
 (cd b && ansible-inventory --list "$@" --output /diff/b.yml)
 
+cd /git
```

Comment:
This feels a little hard-coded, could we set working directory for docker run on the callsite?

Reply:
`WORKDIR` is already set in Dockerfile - the trouble is we've been in about four different directories before we get to this point. Also, it's supposed to be hardcoded, it's a container, we can do what we want ;)